components/lib.ftd

```diff
@@ -0,0 +1,6 @@
+
+-- ftd.background-image bg-image:
+src: $assets.files.assets.bg-back.svg
+repeat: no-repeat
+position: center
+size: contain
```

Comment:
```suggestion
size: contain

```

Always add a new line at the end. Please check this [best practice](https://fastn.com/formatting/#end-line) document.